csharp/src/Drivers/Interop/Snowflake/Apache.Arrow.Adbc.Drivers.Interop.Snowflake.csproj

```diff
@@ -26,13 +36,67 @@
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </Content>
   </ItemGroup>
-  <ItemGroup>
+
+  <!-- What runs in the pipeline. Assumes the Go builds have happened. -->
+  <ItemGroup Condition="$([MSBuild]::IsOSPlatform('Linux'))">
     <Content Include="readme.md">
       <Pack>true</Pack>
       <PackagePath>\</PackagePath>
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </Content>
-  </ItemGroup>
+
+    <!-- Windows/DLLs -->
+    <Content Include="libadbc_driver_snowflake.dll">
+      <Pack>true</Pack>
+      <PackagePath>lib\netstandard2.0</PackagePath>
```

Comment:
Isn't the convention here to put the platform-specific files into e.g. runtimes/win-x64/native, runtimes/mac-arm64/native, etc? This would also avoid needing multiple copies.